src/components/form-elements/TextInput.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+const SIZE = {
+  SMALL: 'small',
+  LARGE: 'large',
+  NORMAL: 'normal'
+};
+
+const COLOR = {
+  NORMAL: 'normal',
+  LIGHT: 'light',
+  LIGHT_ALT: 'light-alt'
+};
+
+const VALIDATION = {
+  VALID: true,
+  INVALID: false,
+  UNDEFINED: undefined
+};
+
+
+const TextInput = props => {
+  const {
+    type = 'text',
+    spacedBottom = true,
+    valid = VALIDATION.UNDEFINED,
+    size = SIZE.NORMAL,
+    color = COLOR.NORMAL,
+    fullWidth,
+    value,
+    ...additionalProps
+  } = props;
+
+  const inputClass = classnames('sg-input', {
+    'sg-input--spaced-bottom': spacedBottom,
```

Comment:
I can delete it from component and create issue in styleguide to delete it from css